wolox-books/src/app/app.component.ts

```diff
@@ -1,10 +1,32 @@
 import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
+
 export class AppComponent {
-  title = 'app';
+  form: FormGroup;
+  post: any[];
+  firstName: string = '';
+  lastName: string = '';
+  email: string = '';
+  password: string = '';
+  locale: string = 'en';
+
+  constructor(private fb: FormBuilder){
+    this.form = fb.group({
+      'firstName': [null, Validators.required],
+      'lastName': [null, Validators.required],
+      'email': [null, Validators.required],
+      'password': [null, Validators.required]
+    })
+  }
+
+  register(post) {
+    post['locale'] = this.locale;
+    console.log(JSON.stringify({'user':post}));
```

Comment:
I think:

 console.log({
  user: {
    ...this.form.value
    locale: "en" 
})

Will do the trick